### PR TITLE
LanguageClient ID. And, use node script to copy yaml file.

### PR DESCRIPTION
**Pull request by sfdctaka**
### What does this PR do?
Implemented feedback from last PR.
* Updated LanguageClient's ID.
* Use node script to determine the platform(Windows vs Unix) where it is running and run appropriate copy command.

### CODE REVIEW

File: scripts/copyComponentExperiencesYaml.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const { execSync } = require('child_process');
+
+const projectRoot = path.resolve(__dirname, '..');
+const outDir = path.join(projectRoot, 'out', 'src', 'resources');
+const sourceFile = path.join(projectRoot, 'resources', 'component-experiences.yaml');
+const destFile = path.join(outDir, 'component-experiences.yaml');
+
+fs.mkdirSync(outDir);
+
+if (process.platform === 'win32') {
```
Comment (khawkins) [inline scripts/copyComponentExperiencesYaml.js:13]: I'd take advantage of Node's file I/O and do away with this conditional as well. I.e.:

```node
fs.copyFileSync(sourceFile, destFile);
```

File: scripts/copyComponentExperiencesYaml.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
```
Comment (khawkins) [inline scripts/copyComponentExperiencesYaml.js:1]: Nit: let's name this a bit more generically, e.g. `copyResources.js`. Opens us up to copying more resources in the future, without renaming the file.

**Comment by khawkins** [comment]
One more change to be made: [line 103](https://github.com/salesforce/salesforcedx-vscode-mobile/pull/161/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R103) of `package.json` needs to be updated from:

```json
        "bundle:extension": "esbuild ./src/extension.ts --bundle --outdir=out --format=cjs --target=es2020 --platform=node --external:vscode --external:@salesforce/core --external:@oclif/core --external:@salesforce/lwc-dev-mobile-core --minify --sourcemap",
```

to:

```json
        "bundle:extension": "esbuild ./src/extension.ts --bundle --outdir=out/src --format=cjs --target=es2020 --platform=node --external:vscode --external:@salesforce/core --external:@oclif/core --external:@salesforce/lwc-dev-mobile-core --minify --sourcemap",
```

`--outdir` needs to be updated to the new output location.

**Comment by sfdctaka** [comment]
Added `package-lock.json` which was supposed to be in the last PR.

**Review by khawkins — APPROVED** [review]

**Pull request merged**